lib/preboots/scheduler.js

```diff
@@ -137,7 +137,16 @@ class Scheduler extends EE {
    * @param {Function} callback - Continuation function to call when completed
    */
   packages(callback) {
-    this.models.PackageCache.findAll({ partitioner: 'cached' }, callback);
+    const results = [];
+
+    this.models.Package.findAll()
```

Comment:
there is a callback version of this too, you just have to give it an empty object as first argument. This works too though :)

Reply:
I'll try to use that, seems cleaner. What I did here is sort of a barebone implementation of the callback logic.